_episodes/10-zonal-statistics.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Calculating Zonal Statistics on Rasters"
+teaching: 40
+exercises: 20
+questions:
+- "How to rasterize a vector data-set?"
+- "How to calculate zonal statistics of a raster data-set?"
```

Comment:
```suggestion
- "How to compute raster statistics on different zones delineated by a vector data?"
```

I changed it a bit because for me when you say over different parts of the vector data, it means the domain is defined by the vector data but it is actually the raster domain.